From c9c90f4f74ee5318ee9a2f581b665d474f6f90c5 Mon Sep 17 00:00:00 2001
From: Stephane Fillod <f8cfe@free.fr>
Date: Tue, 15 Jan 2019 14:56:15 +0100
Subject: [PATCH 7/9] Add various SD/eMMC vendors to table

Part if this list update comes from a compilation of this web site[1].

[1] https://www.cameramemoryspeed.com/sd-memory-card-faq/reading-sd-card-cid-serial-psn-internal-numbers/

Signed-off-by: Stephane Fillod <f8cfe@free.fr>
---
 lsmmc.c | 34 ++++++++++++++++++++++++++++++++--
 1 file changed, 32 insertions(+), 2 deletions(-)

diff --git a/lsmmc.c b/lsmmc.c
index b627b79..63504d2 100644
--- a/lsmmc.c
+++ b/lsmmc.c
@@ -112,7 +112,7 @@ struct ids_database database[] = {
 	{
 		.type = "sd",
 		.id = 0x1b,
-		.manufacturer = "Transcend",
+		.manufacturer = "Transcend/Samsung",
 	},
 	{
 		.type = "sd",
@@ -122,7 +122,7 @@ struct ids_database database[] = {
 	{
 		.type = "sd",
 		.id = 0x1d,
-		.manufacturer = "Corsair",
+		.manufacturer = "Corsair/AData",
 	},
 	{
 		.type = "sd",
@@ -134,6 +134,11 @@ struct ids_database database[] = {
 		.id = 0x1f,
 		.manufacturer = "Kingston",
 	},
+	{
+		.type = "sd",
+		.id = 0x27,
+		.manufacturer = "Delkin/Phison",
+	},
 	{
 		.type = "sd",
 		.id = 0x28,
@@ -144,6 +149,11 @@ struct ids_database database[] = {
 		.id = 0x30,
 		.manufacturer = "SanDisk",
 	},
+	{
+		.type = "sd",
+		.id = 0x31,
+		.manufacturer = "Silicon Power",
+	},
 	{
 		.type = "sd",
 		.id = 0x33,
@@ -159,6 +169,21 @@ struct ids_database database[] = {
 		.id = 0x6f,
 		.manufacturer = "STMicroelectronics",
 	},
+	{
+		.type = "sd",
+		.id = 0x74,
+		.manufacturer = "Transcend",
+	},
+	{
+		.type = "sd",
+		.id = 0x76,
+		.manufacturer = "Patriot",
+	},
+	{
+		.type = "sd",
+		.id = 0x82,
+		.manufacturer = "Gobe/Sony",
+	},
 	{
 		.type = "sd",
 		.id = 0x89,
@@ -224,6 +249,11 @@ struct ids_database database[] = {
 		.id = 0x70,
 		.manufacturer = "Kingston",
 	},
+	{
+		.type = "mmc",
+		.id = 0xfe,
+		.manufacturer = "Micron",
+	},
 };
 
 /* Command line parsing functions */
-- 
2.17.1

